src/context/directory/handlers/index.ts

```diff
@@ -25,13 +25,14 @@ import attackProtection from './attackProtection';
 import branding from './branding';
 
 import DirectoryContext from '..'
+import { AssetTypes } from '../../../types'
 
 export type DirectoryHandler<T> = {
   dump: (context: DirectoryContext) => void,
   parse: (context: DirectoryContext) => T,
 }
 
-export default {
+const directoryHandlers: { [key in AssetTypes]: DirectoryHandler<{ [key: string]: unknown }> } = {
```

Comment:
The addition of `key in AssetTypes` ensures that only and all keys in the object match to strings in the `AssetTypes` union type.